### ISSUE TITLE
have a plugin for being able to `import` an image using ESM

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#517]
## Type of Change
- [x] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
As in #516 , being able to use `import` for images could have some pretty nice DX, e.g.
```js
import logo from '/assets/logo.png';

...

render() {
  return html`<img src="${logo}" alt="logo image"/>
}
```

## Details
Inspired by webpack [**file-loader**](https://webpack.js.org/loaders/file-loader/), this plugin could support / provide metadata too, like:
  - if less then 10KB (for example) return `data://` URI
  - be able to provide dimensions so that you can set width / height in HTML
  - filename hashing
  - other cool stuff
  
I wonder if [`import.meta.url`](https://twitter.com/dassurma/status/1358819452708659204) comes into play here?

**Comment by thescientist13** [comment]
I think let's review this as part of #1163

**Comment by thescientist13** [comment]
This is / can be better solved through #1163  and will be supported in v0.29.0.
